python/apps/taiga/src/taiga/workflows/api/validators.py

```diff
@@ -4,11 +4,19 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 # Copyright (c) 2023-present Kaleidos INC
-
-
+from fastapi import Query
 from pydantic import ConstrainedStr, conint
 from taiga.base.validators import BaseModel
 
+MOVE_TO_DESC = """
```

Comment:
If I understood correctly, this part should be in the API documentation, shouldn't be?

Reply:
I'll remove both the description and the default Query value code